code/controllers/configuration/entries/config.dm

```diff
@@ -348,4 +348,27 @@ CONFIG_DEF(number/error_msg_delay)	// How long to wait between messaging admins
 
 CONFIG_DEF(flag/irc_announce_new_game)
 
-CONFIG_DEF(flag/debug_admin_hrefs)
\ No newline at end of file
+CONFIG_DEF(flag/debug_admin_hrefs)
+
+
+CONFIG_DEF(number/mc_tick_rate/base_mc_tick_rate)
+	integer = FALSE
+	value = 1
+
+CONFIG_DEF(number/mc_tick_rate/high_pop_mc_tick_rate)
+	integer = FALSE
+	value = 1.1
+
+CONFIG_DEF(number/mc_tick_rate/high_pop_mc_mode_amount)
+	value = 65
+
+CONFIG_DEF(number/mc_tick_rate/disable_high_pop_mc_mode_amount)
+	value = 60
+
+CONFIG_TWEAK(number/mc_tick_rate)
+	abstract_type = /datum/config_entry/number/mc_tick_rate
+
+CONFIG_TWEAK(number/mc_tick_rate/ValidateAndSet(str_val))
+	. = ..()
+	if (. && Master)
+		Master.UpdateTickRate()

```

Comment:
Master is created before config so it's probably better to let this runtime if it doesn't exist